man/systemd-nspawn.xml

```diff
@@ -1132,6 +1144,7 @@
       <citerefentry project='mankier'><refentrytitle>dnf</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry project='die-net'><refentrytitle>debootstrap</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry project='archlinux'><refentrytitle>pacman</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+      <ulink url="https://en.opensuse.org/SDB:Zypper_usage">Zypper</ulink>,
```

Comment:
I'd suggest using [the Zypper man page](https://www.mankier.com/8/zypper) instead. The wiki page is out of date...